spec/sidekiq-rate-limiter/server_spec.rb

```diff
@@ -2,19 +2,34 @@
 
 RSpec.describe Sidekiq::RateLimiter, 'server configuration' do
   before do
+    class Job
```

Comment:
This might be better as a local class scoped to the test. Eg:

```
  let(:job_class) do
    Class.new do
      ...
    end
  end
```

I think that would work.

Reply:
Thanks for the local class hint, but it is a pity it doesn't work in this case. Because the Job class needs to be shared between the client and server side, so it can't be a local class.

Reply:
Ah I see - can we move it outside the `Rspec.describe` then?

Reply:
Hmm, let me see if we can do that

Reply:
Thanks, code looks more tidy now